selfdrive/locationd/SConscript

```diff
@@ -7,6 +7,11 @@ if GetOption('kaitai'):
   cmd = f"kaitai-struct-compiler --target cpp_stl --outdir {generated} $SOURCES"
   env.Command(['generated/ubx.cpp', 'generated/ubx.h'], 'ubx.ksy', cmd)
   env.Command(['generated/gps.cpp', 'generated/gps.h'], 'gps.ksy', cmd)
+  env.Command(['generated/glonass.cpp', 'generated/glonass.h'], 'glonass.ksy', cmd)
+
+  # kaitai issue: https://github.com/kaitai-io/kaitai_struct/issues/910
+  kaitai_patch = env.Command(None, 'glonass_fix.patch', 'git apply $SOURCES')
+  env.Depends(kaitai_patch, 'generated/glonass.cpp')
```

Comment:
not sure this works, think you want
```suggestion
  glonass = env.Command(['generated/glonass.cpp', 'generated/glonass.h'], 'glonass.ksy', cmd)

  # kaitai issue: https://github.com/kaitai-io/kaitai_struct/issues/910
  patch = env.Command(None, 'glonass_fix.patch', 'git apply $SOURCES')
  env.Depends(patch, glonass)
```

Reply:
you can also have a dependency on a file